solutions/usaco-787.mdx

```diff
@@ -87,4 +86,96 @@ int main() {
 ```
 
 </CPPSection>
+<JavaSection>
+```java
```

Comment:
use tabs instead of spaces to indent

Reply:
^^ 
> Your entire code uses spaces instead of tabs, see: https://usaco.guide/general/adding-solution#general

Reply:
this hasn't been addressed
https://github.com/cpinitiative/usaco-guide/pull/2030/files#r685643336